docs/spec.md

```diff
@@ -468,9 +468,9 @@ tensor. Depending on the element type, does the following:
 
 #### Inputs
 
-| Name      | Type                                                      | Constraints |
-|-----------|-----------------------------------------------------------|-------------|
-| `operand` | tensor of signed integer, floating-point, or complex type | (C1), (C2)  |
+| Label | Name      | Type                                                      | Constraints |
+|-------|-----------|-----------------------------------------------------------|-------------|
+| (I0)  | `operand` | tensor of signed integer, floating-point, or complex type | (C1), (C2)  |
```

Comment:
Let's start numbering from 1, i.e. `(I1)` etc, to be consistent with constraint labels?

Reply:
done.